src/System.Private.ServiceModel/tools/CertificateGenerator/Program.cs

```diff
@@ -53,10 +53,34 @@ static void UninstallAllCerts()
             RemoveCertificatesFromStore(StoreName.Root, StoreLocation.CurrentUser);
         }
 
-        static void Main(string[] args)
+        static void Usage()
+        {
+            Console.WriteLine("Supported argument is -Uninstall");
+            Console.WriteLine("                      -help");
+        }
+        static int Main(string[] args)
         {
             ApplyAppSettings();
 
+            if (args.Length > 0)
+            {
+                if (string.Compare(args[0], "-Uninstall", true) == 0)
+                {
+                    UninstallAllCerts();
+                    return 0;
+                }
+                else if (string.Compare(args[0], "-help", true) == 0)
+                {
+                    Usage();
+                    return 0;
+                }
+                else
+                {
+                    Usage();
+                    return -1;
```

Comment:
Can we return 1 instead of -1, because script code like `if ERRORLEVEL 1 goto :Failure` will fail to capture failure like this as it can only capture exit code that is equal or greater than 1?